code/modules/asset_cache/assets/rtd.dm

```diff
@@ -0,0 +1,117 @@
+		/obj/item/stack/tile/iron/dark/herringbone,
+		/obj/item/stack/tile/iron/dark_side,
+		/obj/item/stack/tile/iron/dark_corner,
+		/obj/item/stack/tile/iron/checker,
+		/obj/item/stack/tile/iron/dark/textured,
+		/obj/item/stack/tile/iron/dark/textured_edge,
+		/obj/item/stack/tile/iron/dark/textured_half,
+		/obj/item/stack/tile/iron/dark/textured_corner,
+		/obj/item/stack/tile/iron/dark/textured_large,
+		/obj/item/stack/tile/iron/white,
+		/obj/item/stack/tile/iron/white/smooth_edge,
+		/obj/item/stack/tile/iron/white/smooth_half,
+		/obj/item/stack/tile/iron/white/smooth_corner,
+		/obj/item/stack/tile/iron/white/smooth_large,
+		/obj/item/stack/tile/iron/white/small,
+		/obj/item/stack/tile/iron/white/diagonal,
+		/obj/item/stack/tile/iron/white/herringbone,
+		/obj/item/stack/tile/iron/white_side,
+		/obj/item/stack/tile/iron/white_corner,
+		/obj/item/stack/tile/iron/cafeteria,
+		/obj/item/stack/tile/iron/white/textured,
+		/obj/item/stack/tile/iron/white/textured_edge,
+		/obj/item/stack/tile/iron/white/textured_half,
+		/obj/item/stack/tile/iron/white/textured_corner,
+		/obj/item/stack/tile/iron/white/textured_large,
+		/obj/item/stack/tile/iron/recharge_floor,
+		/obj/item/stack/tile/iron/smooth,
+		/obj/item/stack/tile/iron/smooth_edge,
+		/obj/item/stack/tile/iron/smooth_half,
+		/obj/item/stack/tile/iron/smooth_corner,
+		/obj/item/stack/tile/iron/smooth_large,
+		/obj/item/stack/tile/iron/terracotta,
+		/obj/item/stack/tile/iron/terracotta/small,
+		/obj/item/stack/tile/iron/terracotta/diagonal,
+		/obj/item/stack/tile/iron/terracotta/herringbone,
+		/obj/item/stack/tile/iron/kitchen,
+		/obj/item/stack/tile/iron/kitchen/small,
+		/obj/item/stack/tile/iron/kitchen/diagonal,
+		/obj/item/stack/tile/iron/kitchen/herringbone,
+		/obj/item/stack/tile/iron/chapel,
+		/obj/item/stack/tile/iron/showroomfloor,
+		/obj/item/stack/tile/iron/solarpanel,
+		/obj/item/stack/tile/iron/freezer,
+		/obj/item/stack/tile/iron/grimy,
+		/obj/item/stack/tile/iron/sepia,
+		/obj/item/stack/tile/glass,
+		/obj/item/stack/tile/rglass,
+		/obj/item/stack/tile/circuit,
+		/obj/item/stack/tile/circuit/green,
+		/obj/item/stack/tile/circuit/red,
+	)
+
+	for(var/obj/item/stack/tile/tile as anything in tiles)
+		var/icon_state = initial(tile.icon_state)
+		Insert(sprite_name = icon_state , I = 'icons/obj/tiles.dmi', icon_state = icon_state)
+
+		var/list/tile_directions = GLOB.tile_rotations[initial(tile.singular_name)]
+		if(tile_directions == null)
+			continue
+
+		for(var/direction as anything in tile_directions)
+			var/icon/img = icon(icon = 'icons/obj/tiles.dmi', icon_state = icon_state)
+			switch(direction)
+				if(NORTH)
+					img.Turn(-180)
+				if(WEST)
+					img.Turn(90)
+				if(EAST)
+					img.Turn(-90)
+				if(NORTHEAST)
+					img.Turn(-180)
+					var/icon/east_rotated = icon(icon = 'icons/obj/tiles.dmi', icon_state = icon_state)
+					east_rotated.Turn(-90)
+					img.Blend(east_rotated,ICON_MULTIPLY)
+					img.SetIntensity(2,2,2)
```

Comment:
What is SetIntensity?

Reply:
After blending together 2 icons the final image becomes way too dark. So I'm just scaling the r,g,b intensities of the final image so it becomes  a little brighter for the eye

Reply:
Why are we blending multiple icons on top of each other anyway? That seems super whack, I would expect that this would just need to be rotating the tile, not blending multiple things on top of each other.

Reply:
It is wack ><. I can easily oriented image using the image() proc but I can't convert it into an icon for the sprite sheet. 
I don't know what to do

Reply:
Same problem for the below comment